### PR TITLE
Allow the help base url to be configured to update docs links more easily

### PR DESCRIPTION
**Pull request by jwforres**
Fixes https://github.com/openshift/origin-web-console/issues/896

### CODE REVIEW

**Comment by jwforres** [comment]
@spadgett PTAL

**Comment by jwforres** [comment]
actually hold, this isn't doing what i need it to

File: app/scripts/filters/util.js [hunk 1]
```diff
@@ -192,7 +192,11 @@ angular.module('openshiftConsole')
   })
   .filter('helpLink', function(Constants) {
     return function(type) {
-      return Constants.HELP[type] || Constants.HELP["default"];
+      var helpLink = Constants.HELP[type] || Constants.HELP["default"];
+      if (!URI(helpLink).is('absolute')) {
```
Comment (jwforres) [inline app/scripts/filters/util.js]: this is to maintain our existing customization support

**Comment by jwforres** [comment]
@spadgett ok ready now

**Comment by spadgett** [comment]
[merge]

**Pull request merged**